main.py

```diff
@@ -19,6 +19,5 @@ async def lifespan(app: FastAPI):  # pragma: no cover
 app = FastAPI(lifespan=lifespan)
 app.include_router(router)
 
-
-if __name__ == "__main__":  # pragma: no cover
```

Comment:
Why did you remove this comment? I left it not to include this in a coverage report because we cannot test it. Did you check coverage report?